cli/js/testing.ts

```diff
@@ -93,7 +94,7 @@ export function test(
     if (!t) {
       throw new TypeError("The test name can't be empty");
     }
-    testDef = { fn: fn as TestFunction, name: t, ignore: false };
+    testDef = { fn: fn as () => void | Promise<void>, name: t, ignore: false };
```

Comment:
still not sure about removing `TestFunction` - now everywhere you need to write `() => void | Promise<void>`, but let's roll with it for now